package.json

```
@@ -11,17 +11,20 @@
   },
   "scripts": {
     "lerna": "lerna",
-    "build": "lerna run build",
-    "build:docs": "lerna run --scope @refinitiv-ui/docs build --stream",
-    "build:prod": "lerna run build:prod",
+    "build": "nx run-many --target build --parallel --all",
+    "build:affected": "nx affected:build --parallel",
+    "build:docs": "nx run @refinitiv-ui/docs:build",
+    "build:prod": "nx run-many --target build:prod --all",
     "start": "node cli start",
-    "start:docs": "lerna run --scope @refinitiv-ui/docs start --stream",
+    "start:docs": "nx run @refinitiv-ui/docs:start",
     "test": "node cli test",
```

I think adding `reset` command to clear cache would be really useful. 
```suggestion
    "test": "node cli test",
    "reset": "nx reset",
```